bundle/src/main/java/com/adobe/acs/commons/mcp/form/AbstractResourceImpl.java

```diff
@@ -63,6 +63,11 @@ public AbstractResourceImpl(String path, String resourceType, String resourceSup
             meta.put("sling:resourceType", resourceType);
         }
         meta.put(JCR_PRIMARYTYPE, "nt:unstructured");
+        // Required property; Set this to the resource's path
+        meta.put(ResourceMetadata.RESOLUTION_PATH, this.path);
+        // Required property; This information does not exist in this context so set to pl
+        meta.put(ResourceMetadata.RESOLUTION_PATH_INFO, "");
```

Comment:
Since `metadata` can be passed into this function, should we check that `meta.get(ResourceMetadata.RESOLUTION_PATH_INFO)` is `null` prior to defaulting it here?